lib/ui/nav_pages/dm_page/dm_page_viewmodel.dart

```diff
@@ -1,3 +1,11 @@
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/app/app.router.dart';
 import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
 
-class DmPageViewModel extends BaseViewModel {}
+class DmPageViewModel extends BaseViewModel {
+
+  void navigateToDmUser() {
+    locator<NavigationService>().navigateTo(Routes.dmUserView);
```

Comment:
Can kindly make the `NavigationService()` a variable to be use.

`final _navigationService = locator<NavigationService>();`